tests/unit/TodoList1.spec.ts

```diff
@@ -163,13 +163,6 @@ describe('TodoList - UI', () => {
             specialResolution: {}
           }
         }
-      },
-      {
-        'enabled': true,
-        'order': 1,
-        'task': {
-          'filing': { }
-        }
```

Comment:
This is an "empty" filing that caused a console log ("invalid header in filing").